README.md

```diff
@@ -552,7 +552,7 @@ WebAuthProvider.login(account)
 
 #### Accept user invitations
 
-To accept organization invitations your app needs to support [Android App Links](https://developer.android.com/training/app-links). Tapping on the invitation link should open your app (invitations links are `https` only).
+Users can be invited to your organization via a link. Tapping on the invitation link should open your app. Since invitations links are `https` only, is recommended that your app supports [Android App Links](https://developer.android.com/training/app-links). In this [article](https://auth0.com/docs/applications/enable-android-app-links-support) you will find how to make the Auth0 server publish the Digital Asset Links file required by your application.
```

Comment:
Nit: I would encompass more text relevant to the article into the link text, makes it more apparent as to what the read is about to click on.

```suggestion
Users can be invited to your organization via a link. Tapping on the invitation link should open your app. Since invitations links are `https` only, is recommended that your app supports [Android App Links](https://developer.android.com/training/app-links). In [Enable Android App Links Support](https://auth0.com/docs/applications/enable-android-app-links-support), you will find how to make the Auth0 server publish the Digital Asset Links file required by your application.
```